src/Language/AST/NamedTypeNode.php

```diff
@@ -9,6 +9,14 @@ class NamedTypeNode extends Node implements TypeNode
     /** @var string */
     public $kind = NodeKind::NAMED_TYPE;
 
-    /** @var NameNode|string */
+    /** @var NameNode */
     public $name;
+
+    /**
+     * @param mixed[] $edits
+     */
+    public function setEdits(array $edits)
+    {
+        $this->name->value = $edits['name'] ?? $this->name->value;
```

Comment:
The other thing that bothers me is that the incoming name value was never different from the value in `$this->name->value`. In other words, I have no idea what it means to edit this (or any other) node.

Reply:
I understand, now. The only "editing" being done was the printer class's kludgy print method.